src/main/java/dev/shadowsoffire/attributeslib/AttributesLib.java

```diff
@@ -131,6 +132,7 @@ public void setup(FMLCommonSetupEvent e) {
         }
         if (ModList.get().isLoaded("curios")) {
             e.enqueueWork(CuriosCompat::init);
+            MinecraftForge.EVENT_BUS.register(new CuriosClientCompat());
```

Comment:
Client code may not be invoked unconditionally in this file.

Reply:
Agreed. Has been revised and tested to now work on a server.